scripts/host-only/wkdev-create

```diff
@@ -85,6 +85,10 @@ process_command_line_arguments() {
     host_hostname="$(hostnamectl hostname)"
     container_hostname="$(basename "${container_name}").${host_hostname}"
 
+    if argsparse_is_option_set "create-home" && -d "${container_user_home}"; then
+        rm -rf "${container_user_home}"
```

Comment:
I was always afraid about adding an option that does an rm -rf :-)

Reply:
I think we need a counterpart of `wkdev-create` but to remove a container.

Now in the runner there's:

```bash
$ podman ps
CONTAINER ID  IMAGE                            COMMAND               CREATED      STATUS          PORTS       NAMES
97f83258b59f  ghcr.io/igalia/wkdev-sdk:latest  /wkdev-sdk/script...  3 weeks ago  Up 3 weeks ago              wkdev-1727468367
fa1c67dc62f4  ghcr.io/igalia/wkdev-sdk:latest  /wkdev-sdk/script...  3 weeks ago  Up 4 days ago               wkdev
```

For some reason, the temporal container `wkdev-1727468367` was not removed.

Reply:
We usually used podman rm directly. However, I'm not objecting to a wkdev-destroy script.